dom/src/main/scala/fs2/dom/Navigator.scala

```diff
@@ -36,6 +38,8 @@ object Navigator {
 
       def locks = LockManager(navigator.locks)
 
+      def onLine: F[Boolean] = F.pure(navigator.onLine)
```

Comment:
This is a side-effect: it's value may change, so it must be in `delay`.

```suggestion
      def onLine: F[Boolean] = F.delay(navigator.onLine)
```

Reply:
Resolved it